src/separations.cc

```diff
@@ -103,6 +103,10 @@ void Separations::EnterNotify() {
 }
 
 void Separations::Tick() {
+  cyclus::toolkit::RecordTimeSeries<double>(separations_record_demand, this, 
+                                            throughput);
+  cyclus::toolkit::RecordTimeSeries<double>(separations_record_supply, this, 
+                                            throughput);
```

Comment:
I'm not sure it does. We aren't tracking what it actually did that time step, but rather what it is capable of that time step.

Reply:
yeah but processing same amounts of spent fuel may yield different output product quantity depending on the spent fuel composition..right? For example reprocessing 1kg of UNF with 10% plutonium  and 15% plutonium would allow it to output different quantities of pu?

Reply:
Oh yes, true. The supply could be what it actually produces that time step. I still think demand should be based on it's throughput or remaining inventory size.

Reply:
yup! I agree on the demand